### PR TITLE
Explicit white list of allowed extensions, fix #388

### LINKED ISSUE #388: Prevent upload of HTML files as resources

### PR DESCRIPTION
**Pull request by davidbgk**

### CODE REVIEW

**Comment by webnomad** [comment]
Should we really allow all these extensions?

File: js/mixins/uploader.js [hunk 1]
```diff
@@ -31,6 +31,22 @@ define(['api', 'exports?qq!fineuploader'], function(API, qq) {
...
+                validation: {
+                    allowedExtensions: [
+                        // Base
+                        'csv', 'txt', 'json', 'pdf', 'xml', 'rdf', 'epub', 'sql',
+                        // OpenOffice
+                        'ods', 'odt', 'odp', 'odg',
+                        // Microsoft Office
+                        'xls', 'xlsx', 'doc', 'docx', 'pps', 'ppt',
+                        // Archives
+                        'tar', 'gz', 'tgz', 'rar', 'zip', '7z',
```
Comment (noirbizarre) [inline js/mixins/uploader.js]: + .xz

**Comment by noirbizarre** [comment]
r+

**Comment by noirbizarre** [comment]
@webnomad == @vinyll  ??

**Comment by davidbgk** [comment]
I guess so :)

**Comment by vinyll** [comment]
oups, I just revinyllized myself :p

**Pull request merged**